lightly/models/utils.py

```diff
@@ -463,28 +463,33 @@ def random_token_mask(
 
     return idx_keep, idx_mask
 
-def schedule_momentum(
-    iteration: int, 
-    max_iter: int, 
-    m: float = 0.99
+def cosine_decay_schedule(
+    step: int, 
+    max_steps: int, 
+    start_value: float = 0.996,
+    end_value: float = 1
 ) -> float:
 
     """
-    Add sinusoidal noise to momentum value based on iteration number. Common
-    training technique in SSL.
+    Use cosine decay to gradually modify start_value to reach target end_value during training.
+    Introduced in BYOL (https://arxiv.org/pdf/2006.07733.pdf) to update the momentum.
```

Comment:
No need to cite anything here. Cosine schedules are common and existed before BYOL :)